src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -980,6 +997,20 @@ const FormCreationForm = ({formUuid, formUrl, formHistoryUrl}) => {
     });
   }, []);
 
+  useAsync(async () => {
+    // Waiting for the last dispatch to be done to avoid state race conditions.
+    if (!loading) {
+      const responses = await loadFromBackend([
+        {endpoint: FORM_DEFINITIONS_ENDPOINT, query: {is_reusable: true}},
+      ]);
+      const [reusableFormDefinitions] = responses;
+      dispatch({
+        type: 'REUSABLE_FORM_DEFINITIONS_LOADED',
+        payload: reusableFormDefinitions[0],
+      });
```

Comment:
this is still using the `[0]` index? I thought the destructuring would've taken care of that :thinking:

Reply:
woops